app/course/directives/sectionGroupDetails/sectionGroupDetails.js

```diff
@@ -20,6 +20,12 @@ let sectionGroupDetails = function (CourseActionCreators) {
         CourseActionCreators.createSection(section);
       };
 
+      scope.updateSectionGroup = function (sectionGroup) {
+        sectionGroup.unitsVariable ? sectionGroup.unitsVariable = parseFloat(sectionGroup.unitsVariable) : sectionGroup.unitsVariable = null;
```

Comment:
You can rewrite this to be shorter (and avoid repeating yourself 'sectionGroup.unitsVariable =')

sectionGroup.unitsVariable = sectionGroup.unitsVariable ? parseFloat(sectionGroup.unitsVariable) : null;